packages/view/backend/sirius-components-view-emf/src/main/java/org/eclipse/sirius/components/view/emf/diagram/ViewDiagramDescriptionConverter.java

```diff
@@ -231,8 +231,9 @@ private NodeDescription convert(org.eclipse.sirius.components.view.diagram.NodeD
             ILayoutStrategy childrenLayoutStrategy = null;
 
             LayoutStrategyDescription childrenLayoutStrategyFromViewModel = viewNodeDescription.getChildrenLayoutStrategy();
-            if (childrenLayoutStrategyFromViewModel instanceof ListLayoutStrategyDescription) {
-                childrenLayoutStrategy = new ListLayoutStrategy();
+            if (childrenLayoutStrategyFromViewModel instanceof ListLayoutStrategyDescription listLayoutStrategyDescription) {
+                Result result = interpreter.evaluateExpression(variableManager.getVariables(), listLayoutStrategyDescription.getAreChildNodesDraggableExpression());
+                childrenLayoutStrategy = ListLayoutStrategy.newListLayoutStrategy().areChildNodesDraggable(result.asBoolean().orElse(false)).build();
```

Comment:
`ListLayoutStrategy` has `true` as the default value, but here we set `false` if the expression is not set/fails during its evaluation. Is it voluntary?

Reply:
Yes, you're right the default value is true